packages/reports/package.json

```diff
@@ -120,7 +118,6 @@
       "react-router": "^5.2.0",
       "react-router-dom": "^5.2.0",
       "react-transition-group": "^4.3.0",
-      "typescript": "^4.6.3",
```

Comment:
@niloo-fs Why have you removed these dependencies? 🤔

Reply:
Because we have them in the root of the project. we don't need  them in each package

Reply:
@niloo-fs Each package should define its own dependencies regardless of the root.

Reply:
checkout this article https://blog.logrocket.com/setting-up-monorepo-with-lerna-typescript/

Reply:
I'll add them in another PR